GenderPayGap.IdentityServer4/wwwroot/public/assets/govuk_template/assets/stylesheets/govuk-template.css

```diff
@@ -110,7 +110,7 @@ a:active {
  *    www.yuiblog.com/blog/2010/10/01/quick-tip-customizing-the-mobile-safari-tap-highlight-color/
  */
 html {
-  font-size: 62.5%;
+  /*font-size: 62.5%;*/

```

Comment:
I chose to comment these out rather than delete them as there is already clear explanations around what is going on here. So I thought it might make sense to leave it in? But equally happy to remove these.

Reply:
I've seen Aida's changes and it might be best if we drop my changes and rebase on to her work once it gets merged

Reply:
Basically this part of your changes are very similar to what I changed. If you get the review from James first feel free to merge to master and I'll resolve the conflicts

Reply:
I've rebased on top of your work and dropped my commits now. Thanks again for talking me through this bit.